charts/spdk-csi/templates/controller.yaml

```diff
@@ -31,6 +31,7 @@ spec:
         - "--timeout=30s"
         - "--retry-interval-start=500ms"
         - "--leader-election=false"
+        - "--extra-create-metadata=true"
```

Comment:
could you please remind me why was this additional flag added?

Reply:
This field was added so we could get the metadata pvc name and namespace. so we could be able to call it via `request.Getparamter()`

Reply:
Good find. Thanks for clarifying. Great work 💯